project3/styles.css

```diff
@@ -0,0 +1,285 @@
+* {
+    box-sizing: border-box;
+    transition: 400ms;
+}
+
+
+body {
+    background-color: #163346;
+    margin: auto;
+
+}
+
+div {
+    background-color: #1e2019;
+    margin: auto;
+    max-width: 70%;
+    padding: 1.25%;
+
+
+}
+
+fieldset {
+    border: 0;
+}
+
+fieldset span {
+    display: block;
+}
+
+
+
+
+header {
+    background-color: #D1D5DE;
+
+
+}
+
+h1 {
+
+    color: #000000;
+    font-family: 'Teko', sans-serif;
+    font-size: 48px;
+    font-weight: 700;
+    text-transform: uppercase;
+
+}
+
+h2 {
+    color: #fff;
+    font-family: 'Teko', sans-serif;
+    font-size: 40px;
+    font-weight: 500;
+    text-decoration: underline;
+    text-decoration-color: #163346;
+    text-transform: uppercase;
+}
+
+input[type="radio"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 3px solid #eaecf0;
+    border-radius: 50%;
+    cursor: pointer;
+    height: 24px;
+    width: 24px;
+
+}
+
+input[type="checkbox"] {
+    -moz-appearance: none;
+    -webkit-appearance: none;
+    appearance: none;
+    border: 3px solid #eaecf0;
+    cursor: pointer;
+    height: 24px;
+    width: 24px;
+}
+
+input[type="radio"]:checked,
+input[type="checkbox"]:checked {
+    background-color: #71f79f;
+    outline-color: #D1D5DE
+}
+
+input[type="radio"]:checked+label,
+input[type="checkbox"]:checked+label {
+    text-decoration-line: underline;
+    text-decoration-color: #71F79F;
+
+}
+
+input[type="radio"]:hover,
+input[type="checkbox"]:hover {
+    border: 3px solid #71F79F;
+    background-color: #555555;
+
+}
+
+input[type="radio"],
+input[type="checkbox"] {
+    background-color: #1e2019;
+    border: 3px solid #D1D5DE;
+
+
+}
+
+input {
```

Comment:
You probably want these rules to be specific to `input[type="text"], input[type="email"], input[type="number"]`, not all inputs, as it will create conflicts with the radio and checkbox inputs.

Or, if you don't want to get that specific, then this ruleset should appear _before_ the other more specific rules in your CSS file.